sources/packages/api/src/route-controllers/application/application.system.controller.ts

```diff
@@ -243,30 +242,6 @@ export class ApplicationSystemController {
     }
   }
 
-  /**
-   * Updates Confirmation of Enrollment(COE) status.
-   * @param applicationId application id to be updated.
-   * @param payload status of the program information request.
-   */
-  @Patch(":id/coe/status")
-  async updateCOEStatus(
-    @Param("id") applicationId: number,
-    @Body() payload: UpdateCOEStatusDto,
-  ): Promise<void> {
```

Comment:
Can we remove the UpdateCOEStatusDto? in application.system.model

Reply:
Absolutely. Yes.

Reply:
UpdateCOEStatusDto removed.